src/Tracker/index.tsx

```diff
@@ -237,6 +239,56 @@ const setWoopraCookie = (cookie: string): void => {
 
 const getWoopraCookie = (): Optional<string> => (woopra ? woopra.cookie : null)
 
+const cookieAttributes = {
+  name: 'onfido-js-sdk-analytics',
```

Comment:
We have been moving away from using the word JS to refer to the Web SDK. I would rename this to `onfido-web-sdk-analytics`